package.json

```diff
@@ -39,7 +39,9 @@
   "homepage": "https://github.com/MisaOgura/react-express-docker-boilerplate#readme",
   "dependencies": {
     "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "react-dom": "^15.6.1",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2"
```

Comment:
could you also include updated `yarn.lock` file with new dependencies in the PR?